goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
```

Comment:
if we wanted to map SQL to ES semantics, it would be:

index <---> database
doctype <---> table
search <---> query

Maybe it makes sense to start the abstraction up at SavedSearch rather than at AlertSearch with an ESSavedSearch and SQLSavedSearch.